omod/src/main/webapp/pages/results.gsp

```diff
@@ -1,29 +1,34 @@
-<%
-    ui.decorateWith("appui", "standardEmrPage")
-    ui.includeJavascript("uicommons", "datatables/jquery.dataTables.min.js")
-%>
-
+<% ui.decorateWith("appui", "standardEmrPage") %>
+<style>
+.ui-autocomplete {
+    max-height: 100px;
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+</style>
+<script src="//cdn.datatables.net/1.10.15/js/jquery.dataTables.min.js"></script>
```

Comment:
@shivtej1505 Do not use a CDN since sites may not have internet access, you can place the file in omod/resources/scripts and probably use ui.includeJavascript("dataintegrity", "datatables/jquery.dataTables.min.js") - just make sure that you put a version on the js file so that it is clear what version you are using